go.mod

```diff
@@ -3,7 +3,15 @@ module github.com/concourse/s3-resource
 go 1.13
 
 require (
+<<<<<<< HEAD
 	github.com/aws/aws-sdk-go v1.32.0
+=======
+<<<<<<< HEAD
+	github.com/aws/aws-sdk-go v1.34.0
+=======
+	github.com/aws/aws-sdk-go v1.32.0
+>>>>>>> ad6dcd4 (match upstream route to update to 1.16 instead of our own)
+>>>>>>> concourse-master
```

Comment:
This will need to be resolved; if you have a working Go installation locally, you can delete the entire `require() block file and run `go mod tidy`